apps/user-office-frontend-e2e/cypress/e2e/proposals.cy.ts

```diff
@@ -1119,4 +1119,137 @@ context('Proposal tests', () => {
+          createdWorkflowId = result.createProposalWorkflow.id;
+        }
+      });
+      cy.createInstrument(instrument).then((result) => {
+        cy.assignInstrumentToCall({
+          callId: initialDBData.call.id,
+          instrumentIds: result.createInstrument.id,
+        });
+      });
+
+      cy.createTopic({
+        templateId: initialDBData.template.id,
+        sortOrder: 1,
+      }).then((topicResult) => {
+        if (topicResult.createTopic) {
+          const topicId =
+            topicResult.createTopic.steps[
+              topicResult.createTopic.steps.length - 1
+            ].topic.id;
+          cy.createQuestion({
+            categoryId: TemplateCategoryId.PROPOSAL_QUESTIONARY,
+            dataType: DataType.INSTRUMENT_PICKER,
+          }).then((result) => {
+            cy.updateQuestion({
+              id: result.createQuestion.id,
+              question: instrumentPickerQuestion,
+            });
+
+            cy.createQuestionTemplateRelation({
+              questionId: result.createQuestion.id,
+              templateId: initialDBData.template.id,
+              sortOrder: 0,
+              topicId: topicId,
+            });
+          });
+        }
+      });
+    });
+
+    it('Instrument should be automatically assigned to the proposal', () => {
+      cy.login('user1');
+      cy.visit('/');
+
+      cy.contains('New Proposal').click();
+      cy.get('[data-cy=call-list]').find('li:first-child').click();
+
+      cy.get('[data-cy=principal-investigator] input').should(
+        'contain.value',
+        'Carl'
+      );
+
+      cy.finishedLoading();
+
+      cy.contains('New Proposal').click();
+      cy.get('[data-cy=call-list]').find('li:first-child').click();
+
+      cy.get('[data-cy=title] input').type(title).should('have.value', title);
+
+      cy.get('[data-cy=abstract] textarea')
+        .first()
+        .type(abstract)
+        .should('have.value', abstract);
+
+      cy.contains('Save and continue').click();
+
+      cy.get('[role="button"]').first().click();
+
+      cy.get('[role="option"]').contains('Instrument 1').click();
+
+      cy.contains('Save and continue').click();
+
+      cy.finishedLoading();
+
+      cy.notification({ variant: 'success', text: 'Saved' });
+
+      cy.contains('Dashboard').click();
+
+      cy.contains(title).parent().contains('draft');
+
+      cy.contains(title)
+        .parent()
+        .find('[aria-label="Edit proposal"]')
+        .should('exist')
+        .click();
+
+      cy.contains('Submit').click();
+
+      cy.contains('OK').click();
+
+      cy.contains('Dashboard').click();
+      cy.contains(title);
+      cy.contains('submitted');
+
+      cy.get('[aria-label="View proposal"]').should('exist');
+
+      cy.login('officer');
+
+      cy.visit('/');
+
+      cy.contains('Proposals').click();
+
+      cy.contains(title).parent().find('[aria-label="View proposal"]').click();
+
+      cy.contains('td', instrument.name).should('exist');
```

Comment:
As I understand this test case is that you view the proposal and then check if instrument name is the one that you selected in the questionary. This is more like checking the proposal questionary answer. I was wondering if we can check the proposal instrument in the proposals table. If it is auto assigned it should be present even in the table. Something like: `cy.contains(title).parent().contains(instrument.name)`.